src/services/filterUsedAddress.ts

```diff
@@ -9,9 +9,11 @@ import {
   getSpendingKeyHash,
   getAddressesByType,
 } from "../utils";
-import { decode, encode, toWords } from "bech32";
```

Comment:
@pedromtcosta there are a few tests using this package it needs to be updated too.